### PR TITLE
create vicious cycle diarrhea remission rate

### PR DESCRIPTION
**Pull request by rmudambi**
### Description

- *Category*: research model change
- *JIRA issue*: [MIC-2930](https://jira.ihme.washington.edu/browse/MIC-2930)
- *Research reference*: N/A

### Verification and Testing
Ran model

### CODE REVIEW

**Review by albrja — APPROVED** [review]

File: src/vivarium_ciff_sam/constants/data_values.py [hunk 1]
```diff
@@ -96,6 +96,10 @@ class __Wasting(NamedTuple):
         name='value'
     )
 
+    DIARRHEA_DURATION_VICIOUS_CYCLE: Tuple = (
```
Comment (beatrixkh) [inline src/vivarium_ciff_sam/constants/data_values.py:99]: are we expecting this to fix the duration of diarrhea issue?

File: src/vivarium_ciff_sam/constants/data_values.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 # diarrhea duration in days
 DIARRHEA_DURATION: Tuple = (
-    'diarrheal_diseases_duration', get_norm_from_quantiles(mean=4.04485, lower=3.94472, upper=4.144975)
+    'diarrheal_diseases_duration', get_norm_from_quantiles(mean=4.3, lower=4.3, upper=4.3)
```
Comment (beatrixkh) [inline src/vivarium_ciff_sam/constants/data_values.py:21]: This used to have a lower and upper of 4.2 and 4.5, i think. do we want this no-uncertainty distribution? i'm not seeing reference to this on slack

File: src/vivarium_ciff_sam/constants/data_values.py (hunk 1 above)
Comment (rmudambi) [reply src/vivarium_ciff_sam/constants/data_values.py:99]: yes that's the idea. if we calibrate the rr to the actual duration observed in the sim it should be more coherent.

File: src/vivarium_ciff_sam/constants/data_values.py (hunk 2 above)
Comment (rmudambi) [reply src/vivarium_ciff_sam/constants/data_values.py:21]: Hmm they said they wanted 4.3 again, but perhaps they actually wanted the old distribution. Can check in next week if we are doing more runs then

**Pull request merged**